app/src/main/java/com/nextcloud/talk/chat/viewmodels/MessageInputViewModel.kt

```diff
@@ -260,7 +268,19 @@ class MessageInputViewModel @Inject constructor(
                 msg.replyTo!!,
                 msg.sendWithoutNotification!!
             )
+            sleep(DELAY_BETWEEN_QUEUED_MESSAGES)
+        }
+        _messageQueueSizeFlow.tryEmit(0)
+    }
+
+    fun getTempMessagesFromMessageQueue(roomToken: String) {
```

Comment:
just using roomToken is a problem, because we can have multiple accounts on the phone who joined the same conversation (it's an edge case, but it could lead to annoying bugs).

So use the internalId (= accountId@roomToken) for this.